scripts/genproto.sh

```diff
@@ -18,7 +18,7 @@ errcho() {
 
 
 if [ ! -f $protoc ] ; then
-    errcho "Could not find protoc in $protoc. Trying to use protoc in PATH."
+    echo "Could not find protoc in $protoc. Trying to use protoc in PATH."
```

Comment:
I think that leaving it red is better, here's why: if you've run `make install` you should have `protoc` in the propper place. This is probably bugging you (like me) because you're a legacy user. The way to fix it is to re-run `make install`. Why? Because using `protoc` from the `PATH` is error prone, you may not have the correct version, for example. Having this message stand out will make realizing this is the problem faster.

Reply:
Will respond to your related point below